### ISSUE TITLE
IE setupModule Problem

### ISSUE THREAD

**Issue by ibrahimsen** [damienbod/angular-auth-oidc-client#242]
Hello, 

When application starting in IE, I'm getting below error. Is there any idea about it?

![image](https://user-images.githubusercontent.com/2427905/41775513-c92f3d4c-762c-11e8-933a-522c0baa1cec.png)

**Comment by damienbod** [comment]
@ibrahimsen you need to add the npm packages for IE and Edge


https://github.com/damienbod/angular-auth-oidc-client#ie-11-edge-fetch

https://github.com/damienbod/AspNet5IdentityServerAngularImplicitFlow/blob/master/src/AngularClient/angularApp/polyfills.ts#L25
and IE 

https://github.com/damienbod/AspNet5IdentityServerAngularImplicitFlow/blob/master/src/AngularClient/package.json#L43

https://github.com/damienbod/AspNet5IdentityServerAngularImplicitFlow/blob/master/src/AngularClient/angularApp/polyfills.ts#L1

Greetings Damien

**Comment by ibrahimsen** [comment]
Same :(

**Comment by jonasbg** [comment]
I had to put this in the header of my index.html document,
```
 <script>
    /* ref=https://stackoverflow.com/questions/45691804/angular-4-put-a-global-constant-available-to-zone-js/45693219?noredirect=1#comment78345629_45693219 */
    window['__Zone_disable_IE_check'] = true;
  </script>
```

**Comment by FabianGosebrink** [comment]
Is this issue still existing?
